README.adoc

```diff
@@ -2,188 +2,184 @@
+This guide needs three applications.
+The first application (the server application) needs
 only the Eureka Server dependency.
+The second and third applications (the client applications) need Eureka Discovery Client and Spring Web dependencies.
 
-The second application (the client application) needs the Eureka Server and Eureka
-Discovery Client dependencies.
+:server-link: https://start.spring.io/#!type=gradle-project&language=java&packaging=jar&groupId=com.example&artifactId=eureka-server&name=eureka-server&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.eureka-server&dependencies=cloud-eureka-server
+:clienta-link: https://start.spring.io/#!type=gradle-project&language=java&packaging=jar&groupId=com.example&artifactId=servicea&name=servicea&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.servicea&dependencies=cloud-eureka,web
+:clientb-link: https://start.spring.io/#!type=gradle-project&language=java&packaging=jar&groupId=com.example&artifactId=serviceb&name=serviceb&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.serviceb&dependencies=cloud-eureka,web
 
-NOTE: For convenience, we have provided build files (a `pom.xml` file and a `build.gradle`
-file) at the top of the project (one directory above the `service` and `client`
-directories) that you can use to build both projects at once. We also added the Maven and
-Gradle wrappers there.
+You can use the following links for pre-initialized projects from Spring Initializr:
 
-:server-link: https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.5.5&packaging=jar&jvmVersion=11&groupId=com.example&artifactId=service-registration-and-discovery&name=service-registration-and-discovery&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.service-registration-and-discovery&dependencies=cloud-eureka-server
-:client-link: https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.5.5&packaging=jar&jvmVersion=11&groupId=com.example&artifactId=service-registration-and-discovery-client&name=service-registration-and-discovery-client&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.service-registration-and-discovery-client&dependencies=cloud-eureka-server,cloud-eureka
+- {server-link}[Eureka Server^]
+- {clienta-link}[Service A^]
+- {clientb-link}[Service B^]
 
-You can use this {server-link}[pre-initialized project] (for the service application) or this {client-link}[pre-initialized project] (for the client application) and click Generate to download a ZIP file. This project is configured to fit the examples in this tutorial.
+NOTE: Due to the number of services in this guide, only the solutions are provided in the GitHub repository. If you'd like to start from scratch, please use generate the blank projects using the links above or using https://start.spring.io[Spring Initializr] as defined below.
```

Comment:
Replace "if you'd like to" with "To"

Remove "please use"

Add a comma before "as defined below"